### ISSUE TITLE
URL param not working: thirdPerson

### ISSUE THREAD

**Issue by j-conrad** [mozilla/hubs#5094]
**Description**
Using the URL parameter 'thirdPerson' does nothing.

**To Reproduce**
1. Go to any Hubs room
2. Append "**?thirdPerson**" to the URL
3. Reload the page
4. Camera viewpoint stays in 1st person

**Expected behavior**
Camera should be in 3rd person perspective so you can see your own avatar.

**Hardware** 
 - Device: Desktop
 - OS: Windows
 - Browser: Firefox/Chrome/Edge

**Additional context**
This used to work and it listed on [this documentation page.](https://hubs.mozilla.com/docs/hubs-query-string-parameters.html)

**Comment by wsxiaoys** [comment]
Hit key 'o' to switch camera.

**Comment by j-conrad** [comment]
I was just trying this again and discovered that 3rd-person mode is doing **something**, just not the right thing. It appears as though the camera is floating a half a meter or so above my avatar's head. Rotation is odd, but it's unclear what's going on.

**Comment by flankechen** [comment]
@j-conrad 
yes, i dont think the third-person camera is working right in this mode, it facing the wrong directioni of the character
